pyproject.toml

```diff
@@ -7,8 +7,9 @@ build-backend = 'build_backend'
 mypy_path = 'src:test/common'
 exclude = '_vendor'
 [[tool.mypy.overrides]]
-module = ["cockpit._vendor.*"]
+module = ["cockpit._vendor.*", "dbus", "gi.*", "task", "vdo.*"]
 follow_imports = 'silent'
+ignore_missing_imports = true
```

Comment:
Which module requires this, specifically?

Reply:
With reverting this:
```diff
--- pyproject.toml
+++ pyproject.toml
@@ -9,7 +9,6 @@ exclude = '_vendor'
 [[tool.mypy.overrides]]
 module = ["cockpit._vendor.*", "dbus", "gi.*", "task", "vdo.*"]
 follow_imports = 'silent'
-ignore_missing_imports = true
 
 [tool.pylint]
```

these fail:
```
not ok 7 /static-code/test-mypy
# src/client/cockpit-client:30: error: Skipping analyzing "gi": module is installed, but missing library stubs or py.typed marker  [import]
# src/client/cockpit-client:33: error: Skipping analyzing "gi.repository": module is installed, but missing library stubs or py.typed marker  [import]
# src/client/cockpit-client:33: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
# tools/urls-check:30: error: Cannot find implementation or library stub for module named "task"  [import]
# tools/urls-check:30: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
```

So something is special about gi, for "dbus" the `follow_imports=silent` seems to suffice.

This was admittedly a bit of a troll commit, I was hoping you'd reply with "do it like this, bro". I didn't find a way to apply some of the settings to some modules only, something like `[[tool.mypy.overrides.dbus.*]]`.